--- SAL-tarball-v3.5.0-original/ts_ocs_sequencer/bin/setup.env.sh	2017-04-14 00:02:05.000000000 -0500
+++ SAL-tarball-v3.5.0/ts_ocs_sequencer/bin/setup.env.sh	2017-04-13 23:57:45.000000000 -0500
@@ -4,8 +4,15 @@
 # +
 # set the default path below
 # -
-export TS_OCS_SEQUENCER_PATH=/usr/local/ts_ocs_sequencer
 
+if [ -z "TS_OCS_SEQUENCER_PATH" ]; then
+    export TS_OCS_SEQUENCER_PATH=/usr/local/ts_ocs_sequencer
+    echo "Please edit setup.env to set TS_OCS_SEQUENCER_PATH environment variable"
+    echo "to the location where ts_sal and ts_opensplice have been installed."
+    echo "will use default location: $TS_OCS_SEQUENCER_PATH"
+else
+    echo "will use pre-defined location: $TS_OCS_SEQUENCER_PATH"
+fi
 
 # +
 # set derived environmental variable(s)
