--- SAL-tarball-v3.5.0-original/ts_ocs_common/bin/setup.env.sh	2017-04-14 00:02:03.000000000 -0500
+++ SAL-tarball-v3.5.0/ts_ocs_common/bin/setup.env.sh	2017-04-13 23:55:14.000000000 -0500
@@ -3,7 +3,15 @@
 # +
 # set the default path below
 # -
-export TS_OCS_COMMON_PATH=/usr/local/ts_ocs_common
+
+if [ -z "$TS_OCS_COMMON_PATH" ]; then
+    export TS_OCS_COMMON_PATH=/usr/local/ts_ocs_common
+    echo "Please edit setup.env to set TS_OCS_COMMON_PATH environment variable"
+    echo "to the location where ts_sal and ts_opensplice have been installed."
+    echo "will use default location: $TS_OCS_COMMON_PATH"
+else
+    echo "will use pre-defined location: $TS_OCS_COMMON_PATH"
+fi
 
 # +
 # set derived environmental variable(s)
