--- SAL-tarball-v3.5.0-original/ts_sal/setup.env	2017-04-14 00:00:54.000000000 -0500
+++ SAL-tarball-v3.5.0/ts_sal/setup.env	2017-04-14 00:37:01.000000000 -0500
@@ -12,8 +12,8 @@
 if [ -z "$OSPL_HOME" ]; then
     echo "Please edit setup.env to set OSPL_HOME environment variable"
 else
-export SAL_HOME=$LSST_SDK_INSTALL/lsstsal
-export SAL_WORK_DIR=$LSST_SDK_INSTALL/test
+export SAL_HOME=$LSST_SDK_INSTALL/ts_sal/lsstsal
+export SAL_WORK_DIR=$LSST_SDK_INSTALL/ts_sal/test
 export SAL_CPPFLAGS=-m64
 source $SAL_HOME/salenv.sh
 export JAVA_HOME=/etc/alternatives/java_sdk_openjdk
