--- original/setup.env	2017-12-08 11:29:59.000000000 -0600
+++ new/setup.env	2017-12-08 11:52:30.000000000 -0600
@@ -24,8 +24,8 @@
 export PATH=$JAVA_HOME/bin:${SAL_HOME}/bin:${PATH}
 export PYTHONPATH=$PYTHONPATH:${SAL_WORK_DIR}/lib
 export RLM_HOME=$SAL_HOME/.m2/repository/org/opensplice/gateway/rlm/9.1.3
-export PYTHON_BUILD_VERSION=3.6m
-export PYTHON_BUILD_LOCATION=/usr/local
+export PYTHON_BUILD_VERSION=2.7
+export PYTHON_BUILD_LOCATION=/usr
 source $OSPL_HOME/release.com
 mkdir -p $LSST_SDK_INSTALL/lsstsal/lib
 sal_version=`grep -i version $SAL_DIR/sal_version.tcl | awk '{print $3}'`
